README.md

```diff
@@ -15,24 +15,31 @@ Thanks to its strongly typed schema and query language, GraphQL has the potentia
 
 The Apollo GraphQL extension for VS Code brings an all-in-one tooling experience for developing apps with Apollo.
 
-- Add [syntax highlighting](#syntax-highlighting) for GraphQL files and gql templates inside JavaScript files
-- Get instant feedback and [intelligent autocomplete](#intelligent-autocomplete) for fields, arguments, types, and variables as you write queries
-- Manage client-side schema alongside remote schema
-- See [performance information](#performance-insights) inline with your query definitions
-- Extra features to help you with [supergraph editing](#supergraph-editing)
-- Validate field and argument usage in operations
-- [Navigate projects more easily](#navigating-projects) with jump-to and peek-at definitions
-- Manage [client-only](#client-only-schemas) schemas
+The extension provides:
+
+- [Syntax highlighting](#syntax-highlighting) for GraphQL files and `gql` templates in JavaScript
```

Comment:
The aim of this change was to condense all the provided features--the number of bullet points was a little overwhelming.